### ISSUE TITLE
@new update (not an issue, just didnt know where to post) :)

### ISSUE THREAD

**Issue by Joljoh** [Mytherin/Tibialyzer#49]
Is there comming an update from the todays update on Tibia, with the new Ultimate mana potions etc.?
Or is there any way i can add those myself? To calculate to hunt-waste?

**Comment by Mytherin** [comment]
Yes, I will add them in the next update. You can add them to the database yourself as well if you want; just open "Database.db" in SQLite and add the new potions to the "Items" table.

**Comment by taylorkek** [comment]
Tibialyer don't works, is coz the update of tibia?

**Comment by GalLuvton** [comment]
For Mytherin
logStart = 0x53596C
Xor = 0x5356A0
hp = 0x6D3000
MaxHp = 0x6D302C
mp = 0x5356D0
MaxMp = 0x5356A4
I will be working on getting some other addresses (arrows and spears and such).
 can you list what other addresses are needed?

**Comment by Mytherin** [comment]
I already got all the addresses, but thanks for your time! In the next patch I will release in a couple of minutes I have made changing the addresses Tibialyzer uses easier so you don't need to rely on me to update Tibialyzer.

I wonder though, how did you get TabsBaseAddress? I wrote a small program to find it, because finding it manually using Cheat Engine was way too tedious, but maybe there's a better way that I'm not aware of.

**Comment by GalLuvton** [comment]
Its called - CHEATING! ;d
https://github.com/blackdtools/Blackd-Safe-Cheats/blob/master/conf-1092.int
This usually updates within a few hours after an update

**Comment by GalLuvton** [comment]
Note they use the base 400000 address format, so don't just copy-paste stuff from there

**Comment by Mytherin** [comment]
Man, and I spend about two hours finding all these addresses. Oh well. I'll post the update in a few minutes.

**Comment by GalLuvton** [comment]
Like you said before, if someone wanted to find a bot, its very easy to find everything online ;p
Did you get the arrows/boots/stuff addresses? Those I find manually

**Comment by Mytherin** [comment]
Yea I got all the addresses now, thanks for the help.

**Comment by GalLuvton** [comment]
While im talking to you, I messaged you on your reddit account regarding better log reading, utilising the linked list format the log uses. I implemented it in my tool, and it cuts down reading times by a whole bunch (only reading new messages, instead of the whole 1k messages log when its full)

**Comment by Javieracost** [comment]
to @Mytherin  and @GalLuvton, could you share the procedure to get the tabs base address manually? I managed to get all the player related addresses myself the boring way for the linux client, but i need the tabs address to perform a internal message scan similar to Tibialyzer's in [my project](http://github.com/Javieracost/Flarelyzer) ...and it gets complicated without using tools as advanced as CE! Any hint appreciated, and sorry if this is a little offtopic : )

**Comment by Mytherin** [comment]
@GalLuvton Yes I saw your post, good idea. I had been thinking about doing something similar. I haven't had the time to add it in yet but I will in the future.

**Comment by GalLuvton** [comment]
@Javieracost  I did it the super old and boring way, using CE and a lot of backtracking, then I found out that link with the address. Mytherin said he had a program for that, maybe he could help more.
Basically I just searched for 'you are ignoring players', and from there I reversed the procedure, but like I said, I used CE

**Comment by Faceman8787** [comment]
@Mytherin is it possible you could explain how the strcutre of the tabs is build? I have tried alot to get it by myself or by your code but I could not understand it. 
For example: How do I get to the last server log msg starting from your TabsBaseAddress? It would be very helpful to get to know the strcutre of those tabs. Thx.
